### ISSUE TITLE
Login call is taking a long time if user has a large amount of apps

### ISSUE THREAD

**Issue by alexisandreason** [apptreesoftware/revolution#173]
**Description**
The login call is taking an exceptionally long time and making an excessive amount of queries. I'm going to look into what was applied to cut down the configuration fetch time and apply a similar solution to the login call.
**Steps to Reproduce**
Login as apptreesuper to either sandbox 
**App ID**
n/a
**Version Number**
n/a

**Comment by alexisandreason** [comment]
@matthewtsmith Should I mark this as critical?

**Comment by matthewtsmith** [comment]
@alexisandreason yes, but after our next release. I've updated the milestone and labels to account for that.
